index.html

```diff
@@ -2,13 +2,715 @@
+                viewBox="0 0 9 21"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path
+                  d="M.941 19.823 7.1 11.612a2 2 0 0 0 0-2.4L.941 1"
+                  stroke="#383838"
+                  stroke-width="2"
+                />
+              </svg>
+            </div>
+          </div>
+          <div
+            class="block-three__caption block-three__caption--hostel desktop"
+          >
+            Hostel Friendship
+          </div>
+          <div
+            class="block-three__location block-three__location--hostel desktop"
+          >
+            Berlin, Germany
+          </div>
+        </div>
+      </div>
+    </div>
+  <!--Block 4-->
+  <div class="block-four container">
+    <header class="block-four__header center-lg">The best destinations</header>
+    <nav class="block-four__navigation row">
+      <p class="block-four__navigation--p ">Regions</p>
+      <p class="block-four__navigation--p">Cities</p>
+      <p class="block-four__navigation--p desktop">Places of interest</p>
+      <p class="block-four__navigation--p desktop phone">Places</p>
+    </nav>
+    <section class="block-four__content row">
+        <figure class="block-four__content--figure">
+          <img class="block-four__content--figure--img" src="././images/Maldives.png" alt="Maldives">
+          <figcaption class="block-four__content--figure--figcaption">Maldives</figcaption>
+        </figure>
+        <figure class="block-four__content--figure">
+          <div class="block-four__content--figure-block">
+            <img class="block-four__content--figure--img" src="././images/Spain.png" alt="Spain">
+            <div class="block-four--circle phone">
+              <svg
+                class="block-four__content--arrow phone"
+                width="9"
+                height="21"
+                viewBox="0 0 9 21"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path
+                  d="M.941 19.823 7.1 11.612a2 2 0 0 0 0-2.4L.941 1"
+                  stroke="#383838"
+                  stroke-width="2"
+                />
+              </svg>
+            </div>
+          </div>
+          <figcaption class="block-four__content--figure--figcaption">Spain</figcaption>
+        </figure>
+        <figure class="block-four__content--figure desktop">
+          <img class="block-four__content--figure--img desktop" src="././images/Norway.png" alt="Norway">
+          <figcaption class="block-four__content--figure--figcaption desktop">Norway</figcaption>
+        </figure>
+        <figure class="block-four__content--figure desktop">
+          <img class="block-four__content--figure--img desktop" src="././images/island.png" alt="Island">
+          <figcaption class="block-four__content--figure--figcaption desktop">Island</figcaption>
+        </figure>
+    </section>
```

Comment:
It is not the section. The whole "block-four" (terrible name) could be a section.

Reply:
ok. I corrected.